VM/src/lstrlib.cpp

```diff
@@ -1032,6 +1035,33 @@ static int str_format(lua_State* L)
                     break;
                 }
             }
+            case '*':
+            {
+                if (!FFlag::LuauTostringFormatSpecifier)
+                {
+                    if (FFlag::LuauInterpolatedStringBaseSupport)
```

Comment:
This shouldn't be here, as this creates a link-time dependency between VM and higher-level components; we currently don't have a build-time configuration that validates that it doesn't exist, but it would prevent building VM separately without the compiler.

Reply:
How do I lock %* to an FFlag then?

Reply:
Ooooh, I see. The first FFlag is okay because it's declared within the same space, but the second FFlag isn't because it's part of the compiler.